src/main/java/frc/robot/commands/PathPlannerCommand.java

```diff
@@ -0,0 +1,54 @@
+package frc.robot.commands;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+import com.pathplanner.lib.PathConstraints;
+import com.pathplanner.lib.PathPlanner;
+import com.pathplanner.lib.PathPlannerTrajectory;
+import com.pathplanner.lib.PathPoint;
+import com.pathplanner.lib.commands.PPSwerveControllerCommand;
+
+import edu.wpi.first.wpilibj2.command.InstantCommand;
+import edu.wpi.first.wpilibj2.command.PrintCommand;
+import edu.wpi.first.wpilibj2.command.SequentialCommandGroup;
+import frc.robot.constants.AutoConstants;
+import frc.robot.subsystems.Drivetrain;
+import frc.robot.util.PathGroupLoader;
+
+public class PathPlannerCommand extends SequentialCommandGroup {
```

Comment:
I'm pretty sure this is also on a different branch. You're being very ineffiecent if you're working on code we later have to delete because someone else already did it. I understand you need pathplanner stuff for the grid alignment, but the grid alignment should be on a different branch. Make this branch just basic vision, that supports getPose2d(). After that is approved you can have grid alignment, etc.

Reply:
I copied this form RTVision, and we can replace it with a newer version. I can change it to not align to grids yet.

Reply:
Removed to be reimplemented in a later PR